ui/job-view/pushes/Push.jsx

```diff
@@ -150,6 +161,20 @@ class Push extends React.PureComponent {
     return selectedRunnableJobs;
   };
 
+  fetchTestManifests = async () => {
+    // XXX: We might be able to get rid of the backend change if we can use currentRepo
+    const { currentRepo, push } = this.props;
+    // const { revision } = push;
+    const jobTypeNameToManifests = await fetchTestManifests(
+      currentRepo.name,
+      push.revision,
+    );
+    this.setState({ jobTypeNameToManifests });
+    // This causes the jobs to receive the test_path property
+    const { jobList } = this.state;
```

Comment:
By convention, we usually destructure ``state`` at the beginning of the function, like you did for ``props``